src/PositionLink.js

```diff
@@ -5,11 +5,13 @@ import { Link } from 'react-router-dom';
 import { FormattedMessage } from 'react-intl';
 
 import { openRequestStatusFilters } from './utils';
+import { REQUEST_LEVEL_TYPES } from './constants';
 
 export default function PositionLink({ request }) {
   const queuePosition = get(request, 'position');
   const barcode = get(request, 'item.barcode');
-  const openRequestsPath = `/requests?filters=${openRequestStatusFilters}&query=${request.itemId}&sort=Request Date`;
+  const id = request[request.requestLevel === REQUEST_LEVEL_TYPES.ITEM ? 'itemId' : 'instanceId'];
+  const openRequestsPath = `/requests?filters=${openRequestStatusFilters}&query=${id}&sort=Request Date`;
```

Comment:
Could you please move to const `const REQUEST_DATE = 'Request Date';` ?
the same for `export const getURL = (id, count) => <Link to={`/requests?filters=query=${id}&sort=Request Date`}>{count}</Link>;`

Reply:
Done